README.md

```diff
@@ -11,7 +11,7 @@ Together, they allow developers to protect their APIs and web apps without any a
 Some of the features it provides:
 - Transparent login and logout
   - Retrieves OAuth2 Access tokens, ID tokens, and refresh tokens
-- Fine-grained control over which url paths are protected 
+- Fine-grained control over which url paths are protected
```

Comment:
How did you do these format fixes? If you are using any tools, it will be good to include a toolchain.

Reply:
It will be my editor (removing whitespaces), but yeah we have markdown formatter that we can choose.